js/components/graph/Container.js

```diff
@@ -6,11 +6,15 @@ import PropTypes from "prop-types";
 // import * as sidebarDivs from "./sidebar/sidebar_divs.js";
 
 export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.graphRef = React.createRef;
```

Comment:
Nitpick, but change this to `this.graph` so it is following the naming conventions for the refs in `Graph.js`